picframe/start.py

```diff
@@ -124,7 +124,7 @@ def main():
         ]
         check_packages(required_packages)
         print("\nChecking optional packages......")
-        check_packages(['pyheif'])
+        check_packages(['pillow_heif'])
```

Comment:
Wouldn't it be better to make it a required package? 

Then `setup.py`needs to be updated:

````python
    install_requires=[
        'Pillow>=9.0.0',
        'ExifRead',
        'pi3d>=2.49',
        'PyYAML',
        'paho-mqtt',
        'IPTCInfo3',
        'numpy',
        'ninepatch',
        `pillow-heif`
      ],

````